sql/mozfun/glam/map_from_array_offsets_v2/udf.sql

```diff
@@ -0,0 +1,45 @@
+-- udf_get_values
+CREATE OR REPLACE FUNCTION glam.map_from_array_offsets_v2(
```

Comment:
We could call this something more specific to reflect that it help with higher precision. E.g. `map_from_array_offsets_precise`